tests/Unit/HC/reduction_hc.cpp

```diff
@@ -1,594 +1,595 @@
 
```

Comment:
it just cleans up the carriage return ^M character after each /n probably introduced by an editor on Windows